### PR TITLE
PXB-3216 Update Docker docs with information from QuickStart development (8.0)

### PR DESCRIPTION
**Pull request by alina-derkach-oaza**
modified:   docs/docker.md
	modified:   docs/quickstart-docker.md

### CODE REVIEW

File: docs/docker.md [hunk 1]
```diff
@@ -1,80 +1,38 @@
...
-the product. All required libraries are available in
-the container. Being a lightweight execution environment, Docker containers
-enable creating configurations where each program runs in a separate container. You may run
-*Percona Server for MySQL* in one container and *Percona XtraBackup* in
-another. Docker images offer a range of options.
+For this document, container refers to the Docker container and instance refers to the database server in the container.
 
-Create a Docker container based on a Docker image. Docker images for
-Percona XtraBackup are hosted publicly on [Docker Hub](https://hub.docker.com/r/percona/percona-xtrabackup).
+*Percona XtraBackup* has an official Docker image hosted on [Docker Hub](https://hub.docker.com/r/percona/percona-xtrabackup/). If you want the latest version, use the `latest` tag. You can reference a specific version using the [Docker tag filter for the 8.0 versions](https://registry.hub.docker.com/r/percona/percona-xtrabackup/tags?page=1&name=8.0).
```
Comment (patrickbirch) [inline docs/docker.md]: I thought we weren't doing
`*Percona XtraBackup* `

File: docs/docker.md [hunk 2]
```diff
@@ -1,80 +1,38 @@
...
-
-## 1. Install Docker
-
-Your operating system may already provide a package for *docker*. However,
-the versions of Docker provided by your operating system are likely to be
-outdated.
-
-Use the installation instructions for your operating system available from
-the Docker site to set up the latest version of *docker*.
+Make sure that you are using the latest version of Docker. The `apt` and `yum` versions may be outdated and cause errors. Use the [installation instructions](https://docs.docker.com/engine/install/) to install Docker on your system.
```
Comment (patrickbirch) [inline docs/docker.md]: APT and YUM are abbreviations.

File: docs/docker.md [hunk 3]
```diff
@@ -1,80 +1,38 @@
...
-
-## 3. Create a Docker container from Percona XtraBackup image
+## 2. Create Percona XtraBackup Docker container and connect to Percona Server for MySQL container
 
 You can create a Docker container based on Percona XtraBackup image with
 either `docker create` or the `docker run` command. `docker create`
 creates a Docker container and makes it available for starting later.
 
-Docker downloads the Percona XtraBackup image from the Docker Hub. If it
+Docker downloads the Percona XtraBackup 8.0 image from the Docker Hub. If it
```
Comment (patrickbirch) [inline docs/docker.md:23]: ARM64 instructions?

File: docs/docker.md [hunk 4]
```diff
@@ -85,8 +43,13 @@ so that you could easily locate it among your other containers.
 The `volumes-from` flag refers to Percona Server for MySQL and indicates
 that you intend to use the same data as the Percona Server for MySQL container.
 
-Run the container with exactly the same parameters that were used when the
-container was created:
+### Percona XtraBackup ARM64
+
+Starting with Percona XtraBackup 8.0.35-31, you can run the Docker ARM64 version of Percona XtraBackup. You can find the version and architecture on [Docker Hub Percona/Percona-Server Tags](https://registry.hub.docker.com/r/percona/percona-xtrabackup/tags?page=1&name=8.0). The Docker images for ARM64 version contain `aarch64` in the tag name, for example, `8.0.35-aarch64`.
```
Comment (patrickbirch) [inline docs/docker.md:48]: Should add the recommendation that we recommend using the same platform as the server source.

File: docs/quickstart-docker.md [hunk 5]
```diff
@@ -5,93 +5,99 @@ Percona XtraBackup works in combination with a MySQL-compatible database, Percon
...
 {.power-number}
 
-1. Create a Docker volume
+## Create a Docker volume
 
-    One of the benefits of using a Docker volume for persistent storage with a database is that it allows you to keep your data safe and accessible across different containers. For example, if you want to upgrade your database image to a newer version, you can stop the old container and start a new one with the same volume attached without losing any data. Another benefit is that you can quickly backup and restore your data using the docker cp command or mount the volume to another container.
+One of the benefits of using a Docker volume for persistent storage with a database is that it allows you to keep your data safe and accessible across different containers. For example, if you want to upgrade your database image to a newer version, you can stop the old container and start a new one with the same volume attached without losing any data. Another benefit is that you can quickly backup and restore your data using the docker cp command or mount the volume to another container.
```
Comment (patrickbirch) [inline docs/quickstart-docker.md]: Maybe this should be an unordered list? One benefit, another benefit?

File: docs/quickstart-docker.md [hunk 6]
```diff
@@ -5,93 +5,99 @@ Percona XtraBackup works in combination with a MySQL-compatible database, Percon
...
-    |`--password`   | Prompts the user to enter the password for the root user. For convenience you can add the password for the root user to this option, for example, `--password=secret`. Then the password will be passed automaticaly when running the command. Note that if you specify the password with `--password=secret`, the password is visible in `docker ps`, `docker ps -a` (docker history) and regular ps command.          |
-    | `8.0.34` | Use this tag to specify a specific version. Avoid using the `latest` tag. <br> In our example, we use the `8.0.34` tag. In Docker, a tag is a label assigned to an image and is used to maintain different versions of an image.|
+If you do not add a tag, Docker uses `latest` as the default tag and downloads the newest image from [percona/percona-xtrabackup on the Docker Hub](https://hub.docker.com/r/percona/percona-xtrabackup). This image can be in a different series or version from what you expect since the latest image changes over time. The usage of tags ensures that you use [compatible versions](quickstart-overview.md#limitations) of Percona Server for MySQL and Percona XtraBackup.
```
Comment (patrickbirch) [inline docs/quickstart-docker.md]: But if you are using 8.0.34 or later you should be able to make a backup. 
See supported versions.

File: docs/quickstart-docker.md [hunk 7]
```diff
@@ -5,93 +5,99 @@ Percona XtraBackup works in combination with a MySQL-compatible database, Percon
...
 
-        ```{.bash data-prompt="$"}
-        $ docker run --name pxb --volumes-from psmysql -v backupvol:/backup_8034 -it --user root percona/percona-xtrabackup:8.0.34 /bin/bash -c "xtrabackup --backup --datadir=/var/lib/mysql/ --target-dir=/backup_8034 --user=root --password; xtrabackup --prepare --target-dir=/backup_8034"
-        ```
+```{.bash data-prompt="$"}
+$ docker run --name pxb --volumes-from psmysql -v backupvol:/backup_8034 -it --user root percona/percona-xtrabackup:8.0.34 /bin/bash -c "xtrabackup --backup --datadir=/var/lib/mysql/ --target-dir=/backup_8034 --user=root --password; xtrabackup --prepare --target-dir=/backup_8034"
+```
 
-        You are prompted to enter the password, which is `secret`, in our example.
+You are prompted to enter the password, which is `secret`, in our example.
```
Comment (patrickbirch) [inline docs/quickstart-docker.md]: Sentence doesn't make sense.
You are prompted to enter a password. In our example, this password is `secret`.

Comment (alina-derkach-oaza) [reply docs/quickstart-docker.md]: Fixed

File: docs/quickstart-docker.md (hunk 6 above)
Comment (alina-derkach-oaza) [reply docs/quickstart-docker.md]: Updated

File: docs/quickstart-docker.md (hunk 5 above)
Comment (alina-derkach-oaza) [reply docs/quickstart-docker.md]: Done

File: docs/docker.md (hunk 4 above)
Comment (alina-derkach-oaza) [reply docs/docker.md:48]: Added

File: docs/docker.md (hunk 2 above)
Comment (alina-derkach-oaza) [reply docs/docker.md]: fixed

File: docs/docker.md (hunk 1 above)
Comment (alina-derkach-oaza) [reply docs/docker.md]: fixed

**Review by patrickbirch — APPROVED** [review]

**Pull request merged**